frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkSettings.tsx

```diff
@@ -38,7 +36,7 @@ export function EmbeddingSdkSettings({
   const isEmbeddingSdkEnabled = useSetting("enable-embedding-sdk");
 
   function handleChangeSdkOrigins(value: string | null) {
-    updateSetting({ key: SDK_ORIGINS_SETTING.key }, value);
+    updateSetting({ key: sdkOriginsSetting.key }, value);
```

Comment:
I update `useMergeSetting` so the return value has a proper `key` type, so it will work with `updateSetting` by checking the `value` type.